readme.md

```diff
@@ -100,6 +101,38 @@ To run all tests:
 yarn test
 ```
 
+### Development Flow
+
+The tests for the `rps`, `nitro-protocol`, `wallet` and `hub` packages need to interact with a local blockchain.
+
+Run the following script _from the root of the repo_ to start a ganache server, deploy the contracts listed [here](./packages/ganache-deployer/src/deployer.ts), and create a common `NetworkContext` object that holds the relevant contract information for the various packages that need to interface with the corresponding deployed contracts.
```

Comment:
I thought it was important to highlight that (currently) the `ganache-deployer` package actually *deploys* contracts (as its name suggests).